lib/vector-config/src/lib.rs

```diff
@@ -123,7 +101,7 @@
 // attribute on the tuple struct/tuple variant itself to signal that we want to pull the
 // title/description from the field instead, which could be useful when using newtype wrappers
 // around existing/remote types for the purpose of making them `Configurable`.
-#![deny(warnings)]
+#![allow(warnings)]
```

Comment:
Intended switch?

Reply:
Oh, woops!

Meant purely for debugging. I'll revert that.

Reply:
Reverted in aa8d3ba.